__tests__/app.test.js

```diff
@@ -31,8 +32,31 @@ describe("/api/topics", () => {
       .get("/invalid_path")
       .expect(404)
       .then(({ body }) => {
-        console.log(body.msg)
         expect(body.msg).toBe("path not found");
       });
   });
 });
+
+describe("/api", () => {
+      test("200: responds with an object describing all the available endpoints on the API", () => {
+        return request(app)
+          .get("/api")
+          .expect(200)
+          .then(({ body }) => {
+            const endpointKeys = Object.keys(body);
+            expect(endpointKeys[0]).toBe("GET /api");
+            expect(endpointKeys[1]).toBe("GET /api/topics");
+            expect(endpointKeys[2]).toBe("GET /api/articles");
+            const endpoints = [
+              body["GET /api"],
+              body["GET /api/topics"],
+              body["GET /api/articles"],
+            ];
+            endpoints.forEach((endpoint) => {
+                expect(endpoint.hasOwnProperty('description')).toBe(true)
+            })
+          });
+      });
```

Comment:
Nice work here 🎸 you could also require in the endpoints file and assert that it looks the same 🙂